### ISSUE TITLE
FIx  GStreamer-CRITICAL **: gst_element_query_position: assertion 'GST_IS_ELEMENT (element)' failed on exit

### ISSUE THREAD

**Issue by i4004** [i4004/SailPlayer#144]

**Comment by RavinderRikhi** [comment]
I am facing the same issue on my ubuntu. Is it resolved ?

**Comment by i4004** [comment]
Yes, it was resolved but in equalizer branch. This player is for Sailfish OS operating system.

**Comment by RavinderRikhi** [comment]
Okay . I was trying to open up My Webcam feed using OpenCV . The terminal was showing me this error . Can you point me in some direction as I have no clue how to overcome this problem .
